gulpfile.js

```diff
@@ -284,7 +284,7 @@ gulp.task('jsdoc', cb => {
  * Test
  */
 
-gulp.task('test', ['test:unit', 'test:a11y']);
+gulp.task('test', ['test:unit']);
```

Comment:
I guess this is just a temporary change? Please make sure reverting this change before this PR gets out of WIP mode... Thanks!